cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -26,12 +33,20 @@ def _export(dst_file, temp_dir, instance_data, save_images=False):
 
     make_zip_archive(temp_dir, dst_file)
 
-@importer(name='YOLO', ext='ZIP', version='1.1')
-def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+
+def _import_common(
+    src_file,
+    temp_dir,
+    instance_data,
+    load_data_callback=None,
+    format_name="yolo",
```

Comment:
I would suggest removing the default value here and moving it to the import function specific to the original YOLO format. Since this is a common function, it shouldn't have code specific to a particular format.

Reply:
done